src/MorseL.Client/Connection.cs

```diff
@@ -208,7 +208,7 @@ private Task HandleUnparseableReceivedInvocationDescriptor(string serializedInvo
 
             // TODO : Move to a Error type that can be handled specifically
             // Since we don't have an invocation descriptor we can't return an invocation result
-            return _clientWebSocket.SendAsync($"Error: Invalid message \"{serializedInvocationDescriptor}\"");
+            return SendMessageAsync($"Error: Invalid message \"{serializedInvocationDescriptor}\"");
```

Comment:
This is the bug fix for the error messages not passing through the client middleware.